src/tonic/api/points_api.rs

```diff
@@ -137,6 +462,26 @@ impl From<GeoPoint> for segment::types::GeoPoint {
     }
 }
 
+impl From<Range> for segment::types::Range {
+    fn from(value: Range) -> Self {
+        Self {
+            lt: value.lt.map(|v| v.value[0]),   //TODO why payload as vec?
```

Comment:
I have a general issue between the `*Payload` types in gRPC as they contain a vector of values unlike the service types which hold a single value.

How should I deal with that? :thinking:

Reply:
Let's maybe split this type in gPRC: one type should be used for storage (with vector), other - for requests.

Reply:
Done in https://github.com/qdrant/qdrant/pull/264/commits/db8aca3d31fa09bc9e8de6594f0dd96126d150ae